common/static/common/js/discussion/views/discussion_inline_view.js

```diff
@@ -25,6 +25,7 @@
             var match;
 
             this.$el = options.el;
+            this.showThreadPreview = true;
```

Comment:
These changes seem out of place - thread preview text was handled in a separate PR? Is this just showing up here because I haven't rebased `bjacobel/inline-discussion-redesign` on master in a few days?